stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/SheetViewModel.kt

```diff
@@ -67,14 +70,24 @@ internal abstract class SheetViewModel<TransitionTargetType, ViewStateType>(
     }
 
     fun transitionTo(target: TransitionTargetType) {
+        mutableErrorMessage.value = null
         mutableTransition.postValue(target)
     }
 
     fun onError(throwable: Throwable) {
         mutableError.postValue(throwable)
```

Comment:
Should we update the `mutableErrorMessage` here as well? It feels funny to me that we have `mutableError` and `mutableErrorMessage` but they seem to handle different cases.

Reply:
Good observation, I'll rename these variables to clarify the difference.